tests/by-util/test_tail.rs

```diff
@@ -365,61 +379,6 @@ fn test_follow_stdin_name_retry() {
     }
 }
 
-#[test]
-#[cfg(target_os = "linux")]
-#[cfg(disable_until_fixed)]
-fn test_follow_stdin_explicit_indefinitely() {
```

Comment:
I might have asked before, but I forgot. Why was this removed?

Reply:
I've written more tests for this warning than this test would have tested if it were not disabled, so it can be removed in my opinion.